### PR TITLE
add sass and css support to svelte

### PR DESCRIPTION
**Pull request by SaraVieira**
Adds support for (scss) in svelte templates

To test use this example: https://codesandbox.io/s/agitated-snow-u0y0z?file=/App.svelte

### CODE REVIEW

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/4559/commits/663b4a11e231f413de3e92c004ddbf125a7a65e8 is at https://pr4559.build.csb.dev/s/new.

File: packages/app/src/sandbox/eval/presets/svelte/index.js [hunk 1]
```diff
@@ -23,6 +26,23 @@ export default function initialize() {
...
+    { transpiler: styleProcessor },
+    {
+      transpiler: stylesTranspiler,
+      options: { hmrEnabled: false },
+    },
+  ]);
+
+  sveltePreset.registerTranspiler(module => /\.s[c|a]ss$/.test(module.path), [
+    { transpiler: sassTranspiler },
+    { transpiler: styleProcessor },
```
Comment (CompuIves) [inline packages/app/src/sandbox/eval/presets/svelte/index.js:39]: Mini comment: this transpiler can be slow at times, if we don't necessarily need it we should maybe consider not using it, but it's just something to keep in mind.

Comment (SaraVieira) [reply packages/app/src/sandbox/eval/presets/svelte/index.js:39]: I mean I added it because we need it :/

I tried without it

Comment (CompuIves) [reply packages/app/src/sandbox/eval/presets/svelte/index.js:39]: Ah, okay, good to know!

**Pull request merged**